src/main/java/com/interview/solutions/concurrency/ExecutorService.java

```diff
@@ -8,7 +8,7 @@
 import java.util.concurrent.Executors;
 import java.util.concurrent.Future;
 
-public class JthomasExecutorService {
+public class ExecutorService {
```

Comment:
Change the name ExecutorService to something else because java.util.concurrency has an interface named ExecutorService. For readability would be better to change it to your custom ExecutorService name.

Reply:
I will leave those refactoring to you then.